### ISSUE TITLE
Job board considers cargo/bunk capacity for parked escorts

### ISSUE THREAD

**Issue by adamsnd** [endless-sky/endless-sky#100]
Escorts parked (multiple systems away, but that shouldn't matter) still appear to have their available cargo/bunk space count towards the total available cargo/bunk space displayed when browsing "job board" missions. It appears that I cannot actually accept missions that go over my in-system fleet's cargo capacity (tested with only flagship in-system). Did not test against bunk capacity.

Tested with:
* Flagship
  * Bactrian, 90 total cargo, 90 available, no hired crew
* Escorts:
  * 2x Dreadnought, each with 40 total cargo, 40 free, no hired crews

I imagine this section (https://github.com/endless-sky/endless-sky/blob/d09fc4dd89faf7f210c8476525f4a0d56f9d1169/source/MissionPanel.cpp#L539) should include a check against in-system escorts:
```
	int cargoFree = -player.Cargo().Used();
	int bunksFree = -player.Cargo().Passengers();
	for(const shared_ptr<Ship> &ship : player.Ships())
	{
		cargoFree += ship->Attributes().Get("cargo space") - ship->Cargo().Used();
		bunksFree += ship->Attributes().Get("bunks") - ship->Crew() - ship->Cargo().Passengers();
	}
```

**Comment by endless-sky** [comment]
Fixed in commit a41348d. This will be part of the next release (which will be a few weeks away, because I'm about to leave on vacation).

**Comment by endless-sky** [comment]
This is fixed in release 0.8.4.
